src/index.ts

```diff
@@ -877,7 +888,9 @@ export class BufferedChangeset implements IChangeset {
       // 'user.name'
       const normalizedBaseChanges = normalizeObject(baseChanges);
       if (isObject(normalizedBaseChanges)) {
```

Comment:
I was wondering if you should unwrap before the isObject check.

Reply:
Based on your comments before, I was thinking we didn't want to return an `ObjectTreeNode` with a value of `null`.  This might make sense here though.

Reply:
Yes,  ObjectTreeNode with a value of null doesn't make much sense. I was wondering if the whole piece inside the `if (isObject(normalizedBaseChanges))` should be skipped in this case?